### DISCUSSION TITLE
Consejos y buenas prácticas de desarrollo colaborativo

### DISCUSSION THREAD

**Discussion by rcfdtools** [rcfdtools/R.TeachingResearchGuide#8] [General]
https://github.com/rcfdtools/R.TeachingResearchGuide/blob/main/BestPractice.md

A partir de tu propia experiencia, comparte aquí consejos y buenas prácticas de desarrollo colaborativo.

**Comment by juanrodace** [comment]
- Proponte trabajar en tus repositorios un tiempo mínimo por semana, así mantendrás actualizados los contenidos y aumentará la posibilidad de uso y colaboración por parte de otros.
- Habilita el espacio de discusiones en tus repositorios para mejorar la participación y colaboración de tus repositorios.
- Planea desarrollar algún proyecto colaborativo con almenos otra persona y anímense mutuamente a participar activamente.

**Reply by rcfdtools** [reply]
@juanrodace, gracias por tus aportes, en https://github.com/rcfdtools/R.TeachingResearchGuide/blob/main/BestPractice.md podrás visualizar la inclusión de tus consejos y buenas prácticas.
